### PR TITLE
fix: change transak copy for US citizens

### LINKED ISSUE #2051: Change the copy of the onramp to Transak to read "Non-US residents" instead of "international"
Screen Shot 2021-12-07 at 15 34 31

### PR DESCRIPTION
**Pull request by beguene**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/stacks-wallet-web/actions/runs/1553230807).

Fixes #2051 

cc/ @kyranjamie @fbwoolf

### CODE REVIEW

File: src/pages/buy/buy.layout.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { ActiveFiatProviderType } from '@query/hiro-config/hiro-config.query';
 const providersInfo = {
   transak: {
     title: 'Transak',
-    body: 'International users can purchase STX quickly with EUR, KRW and many more.',
+    body: 'Non-US residents users can purchase STX quickly with EUR, KRW and many more.',
```
Comment (kyranjamie) [inline src/pages/buy/buy.layout.tsx]:
```suggestion
    body: 'Non-US residents can purchase STX with credit card, debit card, or bank transfer via Transak.',
```

**Comment by kyranjamie** [comment]
I reckon we merge this straight to `main` and cut a new release.

**Comment by beguene** [comment]
@kyranjamie I will make it a hotfix branch then from main.

**Comment by kyranjamie** [comment]
> @kyranjamie I will make it a hotfix branch then from main.

You can just change the base, right?

**Comment by beguene** [comment]
> > @kyranjamie I will make it a hotfix branch then from main.
> 
> You can just change the base, right?

It will bring all the dev commits, are you fine with that ?

**Comment by kyranjamie** [comment]
Yep, indeed it does. But no, we should remove those extra commits.

**Comment by beguene** [comment]
@kyranjamie Then I will branch off main instead.

File: src/pages/buy/buy.layout.tsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ import { ActiveFiatProviderType } from '@query/hiro-config/hiro-config.query';
 const providersInfo = {
   transak: {
     title: 'Transak',
-    body: 'International users can purchase STX quickly with EUR, KRW and many more.',
+    body: 'Non-US residents can purchase STX quickly with EUR, KRW and many more.',
```
Comment (kyranjamie) [inline src/pages/buy/buy.layout.tsx]:
```suggestion
    body: 'Non-US residents can purchase STX with credit card, debit card, or bank transfer via Transak.',
```

**Review by kyranjamie — APPROVED** [review]

**Comment by beguene** [comment]
@kyranjamie Shall I 'rebase and merge' into main to release now ? I will then merge main back into dev.

**Comment by kyranjamie** [comment]
Yeah, do it 🚢

**Pull request merged**